<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombatDialog</class>
 <widget class="QDialog" name="CombatDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Combat</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="myCharacter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="enemyCharacter">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>380</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="versusImage">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>275</y>
     <width>40</width>
     <height>53</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources/mainwindow.qrc">:/images/combat/versus.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
